lib/brains/webhooks.py

```diff
@@ -30,6 +35,29 @@ def process(self):
 
         log.info('Processing event: {0}'.format(self.webhook.kind))
         method()
+        self.processed = True
+
+    @property
+    def data(self):
+        if not self.processed:
+            return
+
+        subscription = self.get_subscription()
+        try:
+            transaction = self.transactions[0]
```

Comment:
I think somewhere here we should loop through all transactions and log some info about them just so we know what we're missing when we try to figure out how this will work in practice.